releasetool/github.py

```diff
@@ -129,6 +129,7 @@ def create_release(
         target_commitish: str,
         name: str,
         body: str,
+        prerelease: bool = None,
```

Comment:
Could this default to `False` instead?

Reply:
Whoops! That was my intention. I suspect that was a c'n'p error. Will fix now.

Reply:
Done and force-pushed.